jetty-server/src/test/java/org/eclipse/jetty/server/ProxyTestingHelper.java

```diff
@@ -0,0 +1,140 @@
+//
+//  ========================================================================
+//  Copyright (c) 1995-2020 Mort Bay Consulting Pty Ltd and others.
+//  ------------------------------------------------------------------------
+//  All rights reserved. This program and the accompanying materials
+//  are made available under the terms of the Eclipse Public License v1.0
+//  and Apache License v2.0 which accompanies this distribution.
+//
+//      The Eclipse Public License is available at
+//      http://www.eclipse.org/legal/epl-v10.html
+//
+//      The Apache License v2.0 is available at
+//      http://www.opensource.org/licenses/apache2.0.php
+//
+//  You may elect to redistribute this code under either of these licenses.
+//  ========================================================================
+//
+
+package org.eclipse.jetty.server;
+
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
+import java.util.function.Consumer;
+import java.util.stream.Stream;
+
+import org.eclipse.jetty.server.handler.ErrorHandler;
+import org.junit.jupiter.params.provider.Arguments;
+
+public class ProxyTestingHelper
```

Comment:
No helpers for tests, if possible. 
The tests become hard to read because you have to jump to other source files, and they may introduce bugs, and we don't want to write a test framework - tests should be as simple as possible.

Reply:
Fair. Let me find a way to get rid of it.